server/src/resolvers/trainingResolver.ts

```diff
@@ -79,6 +83,70 @@ const TrainingResolvers = {
     deleteOption: async (_: any, args: { id: number }) => {
       await OptionRepo.archiveOption(args.id);
     },
+
+    submitModule: async (
+      parent: any,
+      args: { submission: ModuleSubmissionInput },
+      { ifAllowed }: ApolloContext
+    ) => {
+      return ifAllowed(
+        [Privilege.ADMIN, Privilege.LABBIE, Privilege.MAKER],
+        async (user) => {
+          const submission = args.submission;
+          const answers =
+            await OptionRepo.getCorrectOptionsWithModuleItemByModule(
+              submission.moduleID
+            );
+          if (!answers || answers.length === 0) {
+            throw Error(
+              "Training module for provided ID has no correct answers"
+            );
+          }
+          let correct = 0,
+            incorrect = 0;
+          for (let correctAnswers of answers) {
+            let submittedAnswers = submission.answers.find(
+              (x) => x.moduleItemID == correctAnswers.moduleItemID
+            );
+
+            if (!submittedAnswers) {
+              incorrect++;
+              continue;
+            }
+
+            let areEqual =
```

Comment:
For this variable and the one above, let's prefer `const` over `let` since we're not reassigning them.